messages/de/app.php

```diff
@@ -82,9 +82,9 @@
     'NSFW server' => 'FSK 18 Server',
     'Official Discord Server' => 'Offizieller Discord Server',
     'Official verified partners' => 'Offiziell verifizierte Partner',
-    'Owner servers' => 'Inhaber Server',
+    'Owner servers' => 'Besitzte Server',
```

Comment:
"Besitzte Server" is a grammatically wrong, as the perfect of "besitzen" would be "bessesen".
Both sound weird, so I'm suggesting **"Server in Besitz"**.

I was already tempted to change it to that when I submitted my PR, though I tried to stay as close to "Owner servers" as possible, which is why I changed it to "Inhaber Server".

Reply:
Changed, thank you!

Reply:
Wouldn't **"Deine Server"** work the best?

Reply:
Generally a good idea, but perhaps it would lead to users confusing their owned servers with servers they are on? Also, the tab on the Dashboard for the servers is already called "Deine Server" , so I think keeping a distinction between the tab name and actual owned servers would be good.

Reply:
Yeah, that's true.